resources/wren/shaders/merge_spherical.frag

```diff
@@ -110,20 +112,21 @@ void main() {
     coord.y *= pi_2 / fovY;
 
   vec2 faceCoord = vec2(0.5 * (1.0 - coord.x), 0.5 * (1.0 - coord.y));
```

Comment:
Do we need `faceCoord` and `coord` variables now?

Reply:
`coord` yes as there are several operations on it, `faceCoord` could be indeed dropped by computing directly the index.

Reply:
Sorry, I missed it, it is good as is